mgradm/shared/podman/podman.go

```diff
@@ -47,6 +48,29 @@ func GetExposedPorts(debug bool) []types.PortMap {
 	return ports
 }
 
+// GenerateAttestationSystemdService creates the coco attestation systemd files.
+func GenerateAttestationSystemdService(image string, db install_shared.DbFlags) error {
+	attestationData := templates.AttestationServiceTemplateData{
+		NamePrefix: "uyuni",
+		Network:    podman.UyuniNetwork,
+		Image:      image,
+	}
+	if err := utils.WriteTemplateToFile(attestationData, podman.GetServicePath(podman.ServerAttestationService), 0555, false); err != nil {
+		return fmt.Errorf(L("failed to generate systemd service unit file: %s"), err)
+	}
+
+	environment := fmt.Sprintf(`Environment=UYUNI_IMAGE=%s
+Environment=database_connection=jdbc:postgresql://uyuni-server:%d/%s
```

Comment:
```suggestion
Environment=database_connection=jdbc:postgresql://uyuni-server.mgr.internal:%d/%s
```

We use now a domain for the internal network.

Reply:
Fixed